CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CirrusMD iOS SDK Changelog
 
+# 10.5.0
+
+## Built with: Xcode 14.2.0
+
+### Changes:
+- Adds Pharmacy search module to assessments
+- Removed UDF dependency
```

Comment:
Any other dependency updates or bug fixes worth calling out here? @danrodo @tklucher

Reply:
No, this is the only dependency change for 10.5 all other changes will be in the next release.